plugins/extractors/metabase/metabase_test.go

```diff
@@ -35,18 +34,18 @@ const (
 	dashboard_description  = "some description"
 	email                  = "meteorextractortestuser@gmail.com"
 	pass                   = "meteor_pass_1234"
-	port                   = "3000"
-	url                    = "http://localhost:3000"
+	port                   = "4002"
 )
 
 var (
 	client = &http.Client{
-		Timeout: 2 * time.Second,
+		Timeout: 4 * time.Second,
```

Comment:
was there an issue with 2 secs timeout?

Reply:
Not sure actually, but was getting retry deadline on my device so tried increasing the timeout. Shouldn't be a problem right??

Reply:
hmm 🤔 we may need to look into that, but let's just keep it for now.